Dangerfile

```diff
@@ -37,12 +37,6 @@ if !has_changelog_escape && !git.modified_files.include?("CHANGELOG.md") && has_
     fail("Please include a CHANGELOG entry. \nYou can find it at [CHANGELOG.md](https://github.com/GetStream/stream-chat-swift/blob/master/CHANGELOG.md).")
 end
 
-## Finally, let's combine them and put extra condition 
-## for changed number of lines of code
-if has_app_changes && !has_test_changes && git.lines_of_code > 20
-    warn("Tests were not updated", sticky: false)
-end
-
```

Comment:
Can we comment (instead of removing) and also comment related vars (`has_app_changes`, `has_test_changes`) since they're not used outside of this?

Reply:
I just think this check doesn't make sense. Changes in the production code don't require changes in tests, and vice versa. That's why I removed it.

I liked having `has_app_changes` and `has_test_changes` around for some future use. But I will remove them if you think it makes better sense.

Reply:
I wanted to keep this check as a reminder to add it back when we can (when we have a better testing structure) but the remaining variables are enough reminders (literally `has_test_changes` so I agree, let's remove the check and keep the variables as reminders

Reply:
It lives in git history anyway, we can revert in the future if needed.